### ISSUE TITLE
Undo changes to default port configurations

### ISSUE THREAD

**Issue by jbsbbn** [opennetworkinglab/flowvisor#216]
In FV 0.8, the default configuration for new installs had the XMLRPC interface listening on port 8080. In 1.0, the default configuration changed, to have the JSON interface listening on port 8080. This change breaks clients configured to talk to FV via XMLRPC on port 8080; those clients could be reconfigured, but it'd be simpler if the default configuration for the XMLRPC API were the same (enabled and on port 8080).

**Comment by jbsbbn** [comment]
I tried 1.0.4 just now, and it looks like the XMLRPC API isn't enabled by default:

```
[14:25:57] jbs@jbsofsw-ubuntu:/home/jbs
+$ fvctl --passwd-file=/etc/flowvisor.passwd save-config config.json
Config file written to config.json.

[14:26:18] jbs@jbsofsw-ubuntu:/home/jbs
+$ grep port config.json 
  "api_webserver_port": -1,
  "listen_port": 6633,
  "api_jetty_webserver_port": 8081,
  "controller_port": 0,
```

If you don't want to enable the XMLRPC API by default, we can probably live with that, but since this ticket mentioned "it'd be simpler if the default configuration for the XMLRPC API were the same (enabled and on port 8080)", I wanted to check in case you'd intended to enable it.

**Comment by alshabib** [comment]
Ah yes. I probably just forgot to enable it by default. I'll do that and push another version out. Sorry about that. 

## 

Ali Al-Shabibi (sent from handheld)

**Comment by alshabib** [comment]
fixed 1.0.5 and 50dba9dd2750a7fc6428714e0661063b862d5248
